<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScannerGuiClass</class>
 <widget class="QMainWindow" name="ScannerGuiClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MHY扫码器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pBtLoginAccount">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>添加账号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtstartScreen">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>400</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始监视屏幕</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxAutoScreen">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>510</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>启动时自动监视屏幕</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxAutoExit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>540</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>扫码成功后自动退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtStream">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>400</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始监视直播间</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditUname">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>360</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>未登录</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelUname">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>当前用户名：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelLiveName">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>直播间ID：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLiveId">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>360</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>540</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>v0.0.3</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>510</y>
      <width>118</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>项目源码：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;a href=&quot;https://github.com/Theresa-0328/MHY_Scanner&quot;&gt;Github</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>481</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget 
{
    background:#fbfdfd;
    border:1px solid #bfb1f6;
 
    font-size:13px;                
    font-family:&quot;Microsoft YaHei&quot;;
    color:#37c61a;                
}
 
QTableWidget::item 
{
	border:none;
    border-bottom:1px solid #bfb1f6;
    border-right:1px solid #bfb1f6;
}
 
QTableWidget::item:selected    
{
    color:red;        
    background:#cecbd7;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtSwitch">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>切换账号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtDeleteAccount">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>删除账号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>账号列表</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="action1_3"/>
    <addaction name="action2_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="geometry">
     <rect>
      <x>327</x>
      <y>145</y>
      <width>120</width>
      <height>94</height>
     </rect>
    </property>
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="action1_2"/>
    <addaction name="action2_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="a1">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>反馈问题</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>项目主页</string>
   </property>
  </action>
  <action name="action1_2">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="action2_2">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="action1_3">
   <property name="text">
    <string>软件设置</string>
   </property>
  </action>
  <action name="action2_3">
   <property name="text">
    <string>添加账号</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ScannerGui.qrc"/>
 </resources>
 <connections/>
</ui>
